client/post-editor/editor-drawer/page-options.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getEditedPost } from 'state/posts/selectors';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { getPostType } from 'state/post-types/selectors';
+import PageParent from 'post-editor/editor-page-parent';
+import PageTemplatesData from 'components/data/page-templates-data';
+import PageTemplates from 'post-editor/editor-page-templates';
+import PageOrder from 'post-editor/editor-page-order';
+import Accordion from 'components/accordion';
+import Gridicon from 'components/gridicon';
+
+function EditorDrawerPageOptions( { translate, siteId, postId, post, postType, hierarchical } ) {
+	let title;
+	if ( 'page' === postType ) {
+		title = translate( 'Page Attributes' );
+	} else {
+		title = translate( 'Attributes' );
+	}
+
+	return (
+		<Accordion title={ title } icon={ <Gridicon icon="pages" /> }>
+			{ hierarchical && (
```

Comment:
In the prior implementation, `<PageParent />` and `<PageTemplates />` both had a conditional on `this.props.post` and here they do not.  I tried to see if this would make a difference, but perhaps due to using `connect` here, it didn't seem to cause any issues.